### ISSUE TITLE
Can not login

### ISSUE THREAD

**Issue by askhat** [cptactionhank/docker-netatalk#7]
I've made `docker-compose.yml` in which I specified `AFP_USER` and `AFP_PASSWORD`. `docker inspect` reports that variables are successfully defined. However still can't log in.
![compose](https://cloud.githubusercontent.com/assets/3759947/13415383/5154695a-df6b-11e5-85c2-8bc1d54ff47e.png)
![netatalk](https://cloud.githubusercontent.com/assets/3759947/13415384/515b45d6-df6b-11e5-9ab8-2e7346e21b76.gif)

**Comment by mkkie** [comment]
Check owner & permission on the directory you are sharing

**Comment by askhat** [comment]
@mkkie all set to `askhat:askhat` with the same uid and gid inside the container.
![screen shot 2016-03-02 at 17 30 27](https://cloud.githubusercontent.com/assets/3759947/13463316/aeb5e3e4-e09c-11e5-964a-dfcac18b47de.png)
![screen shot 2016-03-02 at 17 31 30](https://cloud.githubusercontent.com/assets/3759947/13463317/aeb69b5e-e09c-11e5-82b4-2c5bdf77e34e.png)

**Comment by mkkie** [comment]
Check /etc/netatalk/afp.conf  or /etc/afp.conf.
What is the path you are sharing?

[My AFP Volume]
path = /media

You should set the onwer on this folder, and make sure the password is right.
In container :
$ echo askhat:qqwet | chpasswd

The version of cptactionhank/netatalk:latest is 3.1.7.
$ echo $NETATALK_VERSION
3.1.7
I guess the environment variables AFP_USER & AFP_PASSWORD is not working at this version.
You can build your own images from github source code.
$ docker build -t netatalk:3.1.8 .
$ docker run ...... netatalk:3.1.8

**Comment by askhat** [comment]
My `afp.conf` is default, so basically I'm sharing `/media/share` from the
container, with `/mnt/media` from the host, mounted to it. Owner assigned
recursively and double checked, password is same for the host system and
AFP.

**Comment by ghost** [comment]
Hey guys,

I had the same problem. However, I got it to work! Here's how I did it:
`docker exec -it netatalk /bin/bash`
`cat /etc/passwd` doesn't show me in there.
I ran `/docker-entrypoint.sh`
Output:

```
adduser: The GID 1000 does not exist.
chpasswd: (user wiegrafi) pam_chauthtok() failed, error:
Authentication token manipulation error
chpasswd: (line 1, user wiegraf) password not changed
sed: cannot rename /etc/sedbgDotR: Device or resource busy
---begin-afp.conf--
[Global]
; output log entries to stdout instead of syslog
; it is the docker way where the engine in turn
; can direct the log output to a storage backend
log file = /dev/stdout

; enable guest access as well as user accounts
uam list = uams_guest.so uams_dhx2.so uams_dhx.so

[Share]
path = /media/share
; when the environment variable `AFP_USER` is not
; provided valid users list will be empty and
; thus be available for both guests and
; authenticated users
valid users = wiegraf
---end---afp.conf--
mkdir: cannot create directory '/var/run/dbus': File exists
Failed to start message bus: The pid file "/var/run/dbus/pid" exists, if the message bus is not running, remove this file
Daemon already running on PID 19
netatalk is already running (pid = 1), or the lock file is stale.
```

This part is important: `adduser: The GID 1000 does not exist.`
I ran `addgroup --gid 1000 wiegraf` (I happened to know this value beforehand). Then I reran /docker-entrypoint.sh and it worked!

Basically the takeaway is that docker-entrypoint.sh needs to add the group prior to adding the user. Or maybe there is a command to add them both at once - I'm not sure.
